i18n/en.json

```diff
@@ -1046,8 +1046,11 @@
   "admin.plugin.enabling": "Enabling...",
   "admin.plugin.error.activate": "Unable to upload the plugin. It may conflict with another plugin on your server.",
   "admin.plugin.error.extract": "Encountered an error when extracting the plugin. Review your plugin file content and try again.",
+  "admin.plugin.install": "Install",
   "admin.plugin.installedDesc": "Installed plugins on your Mattermost server. Pre-packaged plugins are installed by default, and can be disabled but not removed.",
   "admin.plugin.installedTitle": "Installed Plugins: ",
+  "admin.plugin.installing": "Installing...",
+  "admin.plugin.installTitle": "Install Plugin from URL: ",
```

Comment:
"Install plugin from Secure URL" perhaps to indicate to the user that it IS secure.

Reply:
The download URL entered in the box next to label does not necessarily need to be secure. It only needs to be secure if the setting below is set to false. Does the wording make sense with this in mind?